scitbx/array_family/flex.py

```diff
@@ -576,3 +576,12 @@ def sum(flex_array, axis=None):
       slices[axis] = slice(i,i+1)
       flex_array_sum += flex_array[slices]
     return flex_array_sum
+
+def as_numpy_array(self):
```

Comment:
```suggestion
def _vec3_double_as_numpy_array(flex_array):
```

1. Prefix the function name with an underscore to make it clear it's a private function
2. Rename to make clear that it's specific to `flex.vec3_double`
3. Change the argument from `self` to `flex_array`